catalog/ui/search-ui/standard/src/main/webapp/js/view/filter/FilterCollection.view.js

```diff
@@ -57,7 +57,7 @@ define([
             },
             addChild: function(item){
                 var fieldName = item.get('fieldName');
-                if (fieldName !== Properties.filters.SOURCE_ID && fieldName !== Properties.filters.METADATA_CONTENT_TYPE) {
+                if (fieldName !== Properties.filters.SOURCE_ID && fieldName) {
```

Comment:
If you want the test on `fieldName` being defined, it should probably come first in the condition.